src/iarc7_motion/iarc_tasks/takeoff_task.py

```diff
@@ -129,26 +98,23 @@ def get_desired_command(self):
             if(transStamped.transform.translation.z > self._TAKEOFF_COMPLETE_HEIGHT):
                 self._state = TakeoffTaskState.done
                 return (TaskDone(), NopCommand())
-            elif (self._state == TakeoffTaskState.ascend
-              and transStamped.transform.translation.z
-                > self._ANGLE_MODE_HEIGHT):
-                self._state = TakeoffTaskState.ascend_angle
-                return (TaskRunning(), ArmCommand(True, True, True, lambda _ : None))
+        
```

Comment:
Do we not need to verify that the drone is in angle mode?  Is this done in `fc_comms` now?

Reply:
I don't think we have the check, which is probably the reason why we accidentally took off in rate mode during testing on Thursday. I think that should definitely be implemented into fc_comms

Reply:
fc_comms is handling the angle mode stuff.